app/views/create_new_user_requests/_request_details.html.erb

```diff
@@ -35,17 +35,53 @@
       </span>
     </div>
 
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :access_to_whitehall_publisher, class: "control-label" do %>
+          Does the user need access to Whitehall Publisher?<abbr title="required">*</abbr>
+        <% end %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.collection_radio_buttons :access_to_whitehall_publisher, f.object.access_to_whitehall_publisher_options, :first, :first, required: true, aria: { required: true } do |builder| %>
+          <div class="radio">
+            <%= builder.label class: "custom-control-label" do %>
+              <%= builder.radio_button(class: "custom-control-input") %>
+              <%= builder.object.last %>
+            <% end %>
+          </div>
+        <% end %>
+      </span>
+    </div>
+
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :access_to_other_publishing_apps, class: "control-label" do %>
+          Does this user need access to any other publishing applications?<abbr title="required">*</abbr>
+        <% end %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.collection_radio_buttons :access_to_other_publishing_apps, f.object.access_to_other_publishing_apps_options, :first, :first, required: true, aria: { required: true } do |builder| %>
+          <div class="radio">
+            <%= builder.label class: "custom-control-label" do %>
+              <%= builder.radio_button(class: "custom-control-input") %>
+              <%= builder.object.last %>
+            <% end %>
+          </div>
+        <% end %>
+      </span>
+    </div>
+
     <div class="form-group">
       <span class="form-label">
         <%= f.label :additional_comments do %>
-          What additional publishing permissions does the user need?<abbr title="required">*</abbr>
+          List any other publishing applications and permissions the user needs. If you’re not sure what these are, explain what tasks they need to be able to do.
         <% end %>
       </span>
       <p class="help-block hint-block--md-6" id="additional-comments-hint">
         All users have basic access to Content Data, Feedback Explorer, GovSearch and Support forms.
       </p>
       <span class="form-wrapper">
-        <%= f.text_area :additional_comments, required: true, aria: { required: true }, class: "input-md-6 form-control", rows: 6, cols: 50 %>
```

Comment:
I've asked J-P whether this should be conditional validation based on the answer to the previous quesiton

Reply:
It should, made that change and squashed it into the 4th commit